Scripts/assignment/assignment_period.py

```diff
@@ -476,7 +476,11 @@ def _extract_timecost_from_gcost(self, ass_class):
         gcost = self._get_matrix("gen_cost", ass_class)
         cost = self._get_matrix("cost", ass_class)
         dist = self._get_matrix("dist", ass_class)
-        time = gcost - vot_inv*(cost + self.dist_unit_cost*dist)
+        if ass_class not in ("trailer_truck", "truck"):
```

Comment:
By flipping the `if` and `else`, we could get rid of the `not`, but that is not so important.